fennel/client_tests/test_dataset.py

```diff
@@ -4233,3 +4233,68 @@ def pipeline(cls, event: Dataset):
             [{"value": "4", "ts2": str(now)}, {"value": "3", "ts2": str(now)}],
             [{"value": "5", "ts2": str(now)}],
         ]
+
+
+@mock
+def test_unkey_operator(client):
+    @dataset
+    class JobOpening:
+        creater_id: int
+        job_id: int
+        creation_ts: datetime
+
+    @dataset(index=True)
+    class JobRank:
+        creater_id: int = field(key=True)
+        rank: int = field(key=True)
+        creation_ts: datetime
+
+        @pipeline
+        @inputs(JobOpening)
+        def rank_job(cls, job_opening: Dataset):
+            return (
+                job_opening.groupby("creater_id")
+                .aggregate(
+                    rank=Count(window=Continuous("forever")), emit="final"
+                )
+                .changelog(delete_column="del")
```

Comment:
`delete_column` is quite ugly. What are our alternatives? Could we just call it `delete`?

Reply:
the operator is pretty short, so keeping it explicit wasn't too bad. I mean there are 4 options
del_col
del_column,
delete_col,
delete_column